zilliqa/tests/it/main.rs

```diff
@@ -263,7 +263,7 @@ impl Network {
             consensus: ConsensusConfig {
                 genesis_deposits: genesis_deposits.clone(),
                 is_main: send_to_parent.is_none(),
-                consensus_timeout: Duration::from_secs(1),
+                consensus_timeout: Duration::from_secs(5),
```

Comment:
@bzawisto Why these changes? Will leave them in but an explanation might be useful.